hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -87,12 +87,9 @@ public LocalRegionCache(final String name, final HazelcastInstance hazelcastInst
      */
     public LocalRegionCache(final String name, final HazelcastInstance hazelcastInstance,
                             final CacheDataDescription metadata, final boolean withTopic) {
+        this.name = name;
         this.hazelcastInstance = hazelcastInstance;
-        try {
-            config = hazelcastInstance != null ? hazelcastInstance.getConfig().findMapConfig(name) : null;
-        } catch (UnsupportedOperationException ignored) {
-            EmptyStatement.ignore(ignored);
-        }
+        this.config = lookupMapConfig(name, hazelcastInstance);
```

Comment:
I would have liked to get rid of the `config` field entirely, but it's a `protected` field and so removing it might break something elsewhere. This class itself no longer uses it, though.